src/config.ts

```diff
@@ -354,8 +355,11 @@ function normalizeScripts(cwd: string, scripts: RawScripts): BuildScript[] {
           `scripts[${scriptId}]: "--to ${to}" must be a URL path, and start with a "/"`,
         );
       }
-      const dirDisk = cmdArr[0];
+      let dirDisk = cmdArr[0];
       const dirUrl = to || `/${cmdArr[0]}`;
+      if (scriptId === 'mount:web_modules') {
+        dirDisk = dependenciesLoc;
+      }
```

Comment:
nit: I know the code that was here before was a little confusing (at least to me), and you changed as little as possible. But with this addition do you think you could add a comment or two to explain what this section is doing?

Reply:
Yes! Hadn't seen too many existing comments in the `src` files and had been a bit hesitant to start adding them. Hopefully that's clearer now!